ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -11,7 +11,7 @@ import UIKit
  Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
  */
 @objc(MSFCommandBar)
-open class CommandBar: UIView {
+open class CommandBar: UIView, FluentUIWindowProvider {
```

Comment:
You need to override [didMoveToWindow()](https://developer.apple.com/documentation/uikit/uiview/1622527-didmovetowindow) to update the tokens as they're based on the current window.